tests/unit/Option/NoneTest.php

```diff
@@ -84,6 +84,18 @@ public function testContains(): void
         static::assertFalse($option->contains(4));
     }
 
+    public function testApply(): void
+    {
+        $i = 1;
```

Comment:
Can you use `\Psl\Ref` instead of the PHP reference here?
(it's designed to do just that)